formspree/forms/helpers.py

```diff
@@ -8,7 +8,7 @@
 
 from formspree import settings
 
-HASH = lambda x, y: hashlib.md5(x+y+settings.NONCE_SECRET).hexdigest()
+HASH = lambda x, y: hashlib.md5(x.encode('utf-8')+y.encode('utf-8')+settings.NONCE_SECRET).hexdigest()
```

Comment:
It should be ok. Encoding in Python2 confuses me sometimes, but if that was working, it was because everything was ASCII. If everything continues to be ASCII, encoding to UTF-8 will not change a thing. For non-ASCII stuff, things were probably failing with an horrible exception, but will start working now.

Even the worst happens, I guess the mess will be minimum. It will only affect forms recently created but not yet confirmed.